packages/transport/test/modules/tokens/AnonymousTokenController.test.ts

```diff
@@ -0,0 +1,65 @@
+import { IDatabaseConnection } from "@js-soft/docdb-access-abstractions";
+import { Serializable } from "@js-soft/ts-serval";
+import { CoreDate } from "@nmshd/core-types";
+import { AccountController, AnonymousTokenController, Transport } from "../../../src";
+import { TestUtil } from "../../testHelpers/TestUtil";
+
+describe("AnonymousTokenController", function () {
```

Comment:
you only introduced negative tests here, but no negative ones - not for personalized neither for non personalized tokens.

Reply:
will add a positive test for a non-personalized token, but it's not possible to anonymously load a personalized token

Reply:
right